### DISCUSSION TITLE
How to write to S3 VPC endpoints with AWS Data Wrangler?

### DISCUSSION THREAD

**Discussion by AdrianoNicolucci** [aws/aws-sdk-pandas#621] [Q&A] [answered]
I am trying to run AWS data wrangler on a lambda function that is inside my VPC, I have a VPC endpoint configured in my VPC. AWS Data Wrangler by default attempts to write to s3 over the internet. Is there a way to read or write data with aws data wrangler with VPC endpoints?

**Comment by jaidisido** [comment, marked as answer]
Wrangler does indeed support VPC endpoints for most services covered. The S3 endpoint must be set in your Wrangler global configuration:
- https://aws-data-wrangler.readthedocs.io/en/stable/tutorials/021%20-%20Global%20Configurations.html
- https://github.com/awslabs/aws-data-wrangler/issues/418

**Reply by AdrianoNicolucci** [reply]
This is exactly what I was looking for! Thanks for the link.
